# **Concepts and Standards**
GHGAs metadata model follows several internationally renowned concepts, standards, and resources to provide a metadata schema to share data in a standardized and harmonized fashion.

## **Resources and Standards**

### **FAIR Data Principles**
While digitization is becoming more and more important and technologies accelerate constantly, NGS experiments and measurements produce large quantities of data. Every single dataset in this huge amount of data should be findable and usable for humans and computers equally. In 2016, a conglomerate of representatives of different disciplines - such as academia, industry, funding agencies and scholarly publishers - published the ["FAIR Guiding principles for scientific data management and stewardship"](https://doi.org/10.1038/sdata.2016.18). These principles provide guidance on what to consider when data is published so that an automated and individual exploration, sharing, and reusing of the data is possible. FAIR data should be:  Findable, Accessible, Interoperable and Reusable.

### **FAIRsharing**
Thousands of standards, ontologies and vocabularies have been developed for a variety of communities in order to guide reproducible research. A central database for FAIR standards, repositories and standards is [FAIRsharing](https://fairsharing.org). The mission of the FAIRsharing community is to evaluate standards, databases, policies, and collections. These can be queried by the user’s specific field of interest and can be categorized by Maintained / Not Maintained, Recommended / Not Recommended and Ready / Deprecated / Uncertain / In Dev.

### **Global Alliance for Genomics & Health (GA4GH)**
The [Global Alliance for Genomics and Health (GA4GH)](https://www.ga4gh.org/) is a worldwide acknowledged standards body established to promote globally responsible data sharing of genomic and health-related data. The main objective of this initiative is the alliance of researchers, data scientists, healthcare providers and practitioners and other authorized users while protecting competing interests. GA4GH enables federated data sharing models while preserving the data security, ethical and regulatory framework as well as data authorization and access of sensitive data. Data sharing standards offer data providers the confidence and trust on the data being accessed in accordance with their data policies and without losing control over the multiple downloads of data.

### **RDA alliance**
The [Research Data Alliance (RDA)](https://www.rd-alliance.org/) is a community based initiative formed between the European Commission (EU), the United States Government's National Science Foundation and National Institute of Standards and Technology, and the Australian Government’s Department of Innovation. The main purpose of this initiative is to allow researchers and data providers to practice open data sharing across technologies and geographical locations. Although RDA cannot be considered as a standards development organization (SDO), it fosters building synergies with SDOs and similar organizations so that RDA Recommendations can either be fast-tracked to becoming standards, be incorporated in or contribute to common goals and activities, and overall foster adoption and build trust and engagement in new user communities.

### **Genomics Standards Consortium (GSC)**
The [Genomics Standards Consortium (GSC)]() was one of the earliest launched initiatives towards better descriptions of genomes and metagenomes across communities. The main objective of GSC was to implement new genomic standards, establish methods for capturing and exchanging metadata and harmonization of information across the genomics community. With the support of working groups, GSC has created standards for [Minimum Information about any Sequence (MIxS)]() - which includes genomes (MIGS), metagenomes (MIMS) and environmental genomes (MIMARKS) respectively.

### **Genomic Data Commons**
The [Genomic Data Commons (GDC)](https://datacommons.cancer.gov/repository/genomic-data-commons) was established by the National Cancer Institute (NCI) to boost the understanding of "large-scale, multidimensional data". Therefore the GDC generates datasets to systematize human tumor variations, especially encouraging the unification and sharing of data.  GDC provides the cancer research community with a unified repository and cancer knowledge base that enables data sharing across cancer genomic studies in support of precision medicine. The GDC Data Dictionary is a resource that describes the GDC data model which includes clinical, biospecimen, administrative, and genomic metadata that can be used in parallel with the genomic data generated by the GDC.
The properties and the values in the GDC data dictionary contain references to external standards which are defined and maintained by [NCI Thesaurus (NCIt)](https://ncithesaurus.nci.nih.gov/ncitbrowser/) and the [Cancer Data Standards Registry and Repository (caDSR)](https://datascience.cancer.gov/resources/metadata).

## **Metadata standards**
Metadata provides context and provenance to raw data and methods and are essential to both discovery and validation. It can be classified as a high level document which establishes a common way of structuring and understanding data by including principles and implementation issues utilizing the standard. Metadata standards offer conventions for the generation and description of research data. They specify and define the structure of metadata.

### **DublinCore (DC) metadata element set**
The [Dublin Core™ Metadata Element Set](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) is a vocabulary of fifteen properties for use in resource description. The fifteen element "Dublin Core™" described in this standard is part of a larger set of metadata vocabularies and technical specifications maintained by the [Dublin Core™ Metadata Initiative (DCMI)](https://www.dublincore.org/). The full set of vocabularies, DCMI Metadata Terms \[DCMI-TERMS\], also includes sets of resource classes (including the DCMI Type Vocabulary \[DCMI-TYPE\]), vocabulary encoding schemes, and syntax encoding schemes. The terms in DCMI vocabularies are intended to be used in combination with terms from other, compatible vocabularies in the context of application profiles and on the basis of the DCMI Abstract Model \[DCAM\].

### **Minimum Information (MI) Standards**
[Minimum information (MI) standards](https://www.ebi.ac.uk/training/online/courses/bioinformatics-terrified/what-makes-a-good-bioinformatics-database/minimum-information-standards/) are sets of guidelines and formats for reporting data derived by specific high-throughput methods. The purpose of MI standards is to ensure the data generated by high-throughput methods can be easily verified, analyzed and interpreted by the wider scientific community.  Minimal information standards are available for a vast variety of experiment types including microarray ([MIAME](https://fairsharing.org/FAIRsharing.32b10v)), RNAseq ([MINSEQE](https://fairsharing.org/FAIRsharing.a55z32)), metabolomics ([MSI](https://github.com/MSI-Metabolomics-Standards-Initiative/CIMR)) and proteomics ([MIAPE](https://www.psidev.info/miape)).

### **Minimum information about a Microarray Experiment (MIAME)**
The [MIAME](https://fairsharing.org/FAIRsharing.32b10v) guidelines were created by the [Functional Genomics Data Society (FGED)](https://www.fged.org/) society to describe the standards for recording and reporting microarray-based gene expression data to facilitate data access and interpretation from public repositories such as [Gene Expression Omnibus (GEO)](https://www.ncbi.nlm.nih.gov/geo/).

### **Minimum Information about a Proteomic Experiment (MIAPE)**
The [MIAPE](https://www.psidev.info/miape) guidelines were founded by the [Human Proteome Organization's Proteomics Standards Initiative](https://www.hupo.org/Proteomics-Standards-Initiative-(PSI)) to develop guidelines for reporting the use of techniques such as gel electrophoresis and mass spectrometry.  One of the main objectives of the MIAPE is to increase the value derived by the scientific community from ongoing experimentation in proteomics. This is established through community processes that support sharing, dissemination and reanalysis of datasets, and that assist in establishing and promoting best practice in specific technical areas. The most important MI standards for genomics are listed below.

### **Minimum Information about a high-throughput Nucleotide Sequencing Experiment (MINSEQE)**
[MINSEQE](https://fairsharing.org/FAIRsharing.a55z32) describes the minimum information about a high-throughput nucleotide sequencing experiment that is needed to enable the unambiguous interpretation and facilitate reproduction of the results of the experiment. By analogy to the [MIAME](https://fairsharing.org/FAIRsharing.32b10v) guidelines for microarray experiments, adherence to the [MINSEQE](https://fairsharing.org/FAIRsharing.a55z32) guidelines will improve integration of multiple experiments across different modalities, thereby maximizing the value of high-throughput research. The five main elements of experimental description to be [MINSEQUE](https://fairsharing.org/FAIRsharing.a55z32) compliant include - description of the experiment and sample under study, sequence read data for each assay, final processed data for the study, information about experiment-sample relationship, experiment and sample processing protocol.


## **Ontologies**

To ensure that the metadata that is collected in GHGA is of high quality, we support a selection of ontologies for certain properties where their values can be one or more concept terms from these ontologies. The ontologies were chosen based on their suitability to represent the knowledge specific to genomic medicine. They have a wide adoption and community support, which increases their interoperability and reusability.

### **Bioscientific data analysis ontology**
[Bioscientific data analysis ontology (EDAM)](https://bioportal.bioontology.org/ontologies/EDAM) is a straightforward ontology that consists of commonly known and widely used concepts in the field of bioinformatics, such as data types and identifiers, data formats, operations, and topics. It offers a collection of terms that come with definitions and synonyms, all organized into an easily understandable hierarchy for convenient usage. We recommend the use of concepts from EDAM to represent file formats. For example, instead of using free text ‘FASTQ’ to represent a file in FASTQ-format, we recommend using the appropriate concept **Format:1930 FASTQ**.

### **BRENDA Tissue Ontology**
The [BRENDA Tissue Ontology (BTO)](https://obofoundry.org/ontology/bto.html ) provides a structured controlled vocabulary to describe the source of an enzyme. The ontology contains terms to represent tissues, cell lines, cell types and cell cultures. These terms span uni- and multicellular organisms. We recommend the use of concepts from BTO to represent anatomical location/site associated with a Biospecimen and/or a Sample. For example, instead of using free text ‘heart tissue’ to represent the site from which a Biospecimen was derived from, we would recommend using the appropriate concept **BTO:0004293 heart endothelium**.

### **Data Use Ontology**
Endorsed by GA4GH, the [Data Use Ontology (DUO)](https://obofoundry.org/ontology/duo.html) allows users to tag datasets with usage restrictions, allowing them to become automatically discoverable based on a health, clinical, or biomedical researcher’s authorization level or intended use. We recommend the use of concepts from DUO to represent the use restrictions associated with a Dataset. For example, instead of having use restrictions as free text in a Data Access Policy, we would recommend using the appropriate concepts from DUO to better represent the granularity of use conditions and restrictions.

### **Experimental Factor Ontology**
The [Experimental Factor Ontology (EFO)](https://www.ebi.ac.uk/efo) provides a systematic description of many experimental variables available in databases like those from the EBI. EFO combines parts of several biological ontologies, such as UBERON anatomy, ChEBI chemical compounds, and Cell Ontology. EFO is endorsed by [EMBL-EBI](https://www.ebi.ac.uk/services), [EGA](https://ega-archive.org), and [ENA](https://www.ebi.ac.uk/ena/browser). We recommend the use of concepts from EFO to represent experimental factors that are typically associated with studies. For example, instead of using free text ‘Exome sequencing’ to signify the type of an Experiment, we would recommend using the appropriate concept **EFO:0005396 Exome sequencing**.

### **The Gender, Sex, and Sexual Orientation Ontology**
The [Gender, Sex, and Sexual Orientation Ontology (GSSO)](https://obofoundry.org/ontology/gsso.html) offers terms to describe gender, sex, and sexual orientation. It is aimed at interdisciplinary research in the biomedical and related sciences. We recommend the use of concepts from GSSO to represent the biological sex of an individual. For example, instead of using free text ‘female’ to represent the sex of an individual, we would recommend using the appropriate concept **GSSO:011317 female sex for clinical use**.

### **Human Ancestry Ontology**
The [Human Ancestry Ontology (HANCESTRO)](https://obofoundry.org/ontology/hancestro) provides a systematic description of the ancestry concepts. HANCESTRO was originally built for NHGRI-GWAS Catalog and has since then been used by other consortia like the GA4GH, and the [Human Cell Atlas](https://www.humancellatlas.org). We recommend the use of concepts from HANCESTRO to represent the ancestry of an Individual. For example, instead of using ‘European ancestry’ to represent the ancestry of an Individual, we would recommend using the appropriate concept **HANCESTRO:0005 European**.

### **Human Phenotype Ontology**
The [Human Phenotype Ontology (HPO)](https://obofoundry.org/ontology/hp) provides a standardized vocabulary of phenotypic abnormalities encountered in human disease. HPO is used by various consortia like the [GA4GH](https://www.ga4gh.org), [Solve-RD](https://solve-rd.eu), and [IRDiRC](https://irdirc.org). We recommend the use of concepts from HPO to represent phenotypic abnormalities that characterize a Biospecimen and/or an Individual. For example, instead of using free text ‘Heart attack’ to represent an Individual who has suffered from a heart attack, we would recommend using the appropriate concept **HP:0001658 Myocardial infarction**.

### **International Classification of Diseases**
The [International Classification of Diseases (ICD)](https://www.who.int/classifications/classification-of-diseases) is widely used across the world and is a crucial source of information on the prevalence, causes, and outcomes of human disease and mortality. Through the use of standardized coding, clinical information can be collected and recorded using ICD in primary, secondary, and tertiary care settings, as well as on death certificates. These data form the foundation for disease surveillance and statistical analysis, which inform healthcare planning, payment systems, quality control, and research. In addition, ICD's diagnostic categories facilitate consistent data collection and enable large-scale research studies. We recommend the use of classifications from ICD to represent a diagnosis associated with an Individual. For example, instead of using free text ‘Malignant neoplasm of thymus’ to represent that an Individual suffers from thymic carcinoma, we would recommend using the appropriate concept **C37 Malignant neoplasm of thymus**.

### **Mondo Disease Ontology**
The [Mondo Disease Ontology (Mondo)](https://obofoundry.org/ontology/mondo) provides a unified disease terminology that yields precise equivalences between disease concepts across various terminologies like OMIM, Orphanet, EFO, and DOID. Mondo is used by several consortia like GA4GH, [ClinGen](https://clinicalgenome.org), and [Gabriella Miller Kids First](https://kidsfirstdrc.org). We recommend the use of concepts from Mondo to represent diseases associated with a Biospecimen and/or an Individual. For example, instead of using free text ‘Myocardial infarction’ to represent an Individual who has suffered from a heart attack, we would recommend using the appropriate concept **MONDO:0005068 Myocardial infarction**.

### **National Cancer Institute Thesaurus**
The [National Cancer Institute Thesaurus (NCIt)](https://obofoundry.org/ontology/ncit.html ) is a reference terminology covering the cancer domain, including diseases, abnormalities, anatomy, drugs, genes, and more. It provides granular and consistent terminology in certain areas like cancer diseases and combination chemotherapies. The terminology is a combination from numerous cancer research domains and enables integration of information through semantic relationships. We recommend the use of concepts from NCIt to represent the case or control status associated with a Sample. For example, instead of using free text ‘True Case Status’ to represent the case status of a sample, we would recommend using the appropriate concept **NCIT:C99269 True Case Status**.

### **Orphanet Rare Disease Ontology**
Orphanet and the EBI have collaborated to develop the [Orphanet Rare Disease Ontology (ORDO)](https://www.ebi.ac.uk/ols/ontologies/ordo), which provides a well-organized and structured vocabulary for rare diseases. This ontology captures the relationships between diseases, genes, and other relevant features, and it serves as a valuable resource for the computational analysis of rare diseases. The Orphanet database, which is a multilingual database dedicated to rare diseases that is populated from literature and validated by international experts, serves as the basis for the ORDO. The ORDO incorporates a nosology, which is a classification system for rare diseases, as well as relationships such as gene-disease connections and epidemiological data. Additionally, the ORDO is connected to other terminologies like [Medical Subject Headings (MeSH)](https://www.nlm.nih.gov/mesh/meshhome.html), [Unified Medical Language System (UMLS)](https://www.nlm.nih.gov/research/umls/index.html), and [Medical Dictionary for Regulatory Activities (MedDRA)](https://www.ich.org/page/meddra), databases like [OMIM](https://www.omim.org/), [UniProtKB](https://www.uniprot.org/), [HGNC](https://www.genenames.org/), [Ensembl](https://www.ensembl.org/index.html), [Reactome](https://reactome.org/), The [International Union of Basic and Clinical Pharmacology (IUPHAR)](https://www.guidetopharmacology.org/), [GenAtlas](https://bio.tools/genatlas), and classifications like ICD10. We recommend the use of concepts from ORDO to represent phenotypic features that characterize a Biospecimen and/or Individual. For example, instead of using free text ‘Duchenne muscular dystrophy’ to represent an Individual with Duchenne, we recommend using the appropriate concept **ORPHA:98896 Duchenne muscular dystrophy**.

### **Systematic Nomenclature of Medicine Clinical Terms**
[SNOMED Clinical Terms (SNOMED CT)](https://www.ebi.ac.uk/ols/ontologies/snomed) is a computerized repository of medical terms that are systematically organized for easy processing. This collection includes codes, terms, synonyms, and definitions that are commonly used in clinical documentation and reporting. We recommend the use of concepts from SNOMED CT to represent a sampling method associated with a Biospecimen and/or a Sample. For example, instead of using free text ‘Bone marrow sampling’ to represent the method used to isolate a sample, we would recommend using the appropriate concept **SNOMEDCT:234326005 Bone marrow sampling**.

### **Semanticscience Integrated Ontology**
The [Semanticscience Integrated Ontology (SIO)](https://doi.org/10.1186/2041-1480-5-14) is a biomedical ontology for knowledge discovery. It describes diverse objects, processes, and attributes (real or hypothetical) using simple design patterns. SIO has extensions for chemistry, biology, biochemistry, and bioinformatics. It underpins the Bio2RDF linked data project and aids semantic integration for SADI-based web services. To unambiguously indicate the meaning of properties in the GHGA Metadata Schema they are linked to SIO terms, e.g. **SIO:000089 dataset**.

### **Uber-Anatomy Ontology**
[Uber-Anatomy Ontology (Uberon)](http://obophenotype.github.io/uberon/) is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. Uberon is used in various databases like ENA, EGA, and EBI [BioSamples](https://www.ebi.ac.uk/biosamples). We recommend the use of concepts from Uberon to represent anatomical location/site associated with a Biospecimen and/or a Sample. For example, instead of using free text ‘heart tissue’ to represent the site from which a Biospecimen was derived from, we would recommend using the appropriate concept **UBERON:0008307 heart endothelium**.
